docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5933` FTX and FTX US are no longer supported.
```

Comment:
Will probably add this in a subsequent PR when I touch the changelog. Or you can do it too.

The way you wrote it, sounded as if we removed support unilaterally.

```suggestion
* :feature:`5933` FTX and FTX US support is removed since the exchanges no longer exist.
```